repo/mpi-pingpong/package.py

```diff
@@ -0,0 +1,67 @@
+# Copyright 2023 Lawrence Livermore National Security, LLC and other
+# Benchpark Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+from spack.package import *
+
+
+class MpiPingpong(CMakePackage, ROCmPackage, CudaPackage):
+
+    git = "https://github.com/LLNL/microbenchmarks.git"
+
+    version("develop", branch="develop")
+
+    variant("caliper", default=False, description="Enable Caliper/Adiak support")
+    variant("rocm", default=False, description="Enable Rocm support")
+    variant("cuda", default=False, description="Enable CUDA support")
+    variant("mpi", default=True, description="Enable MPI support")
+
+    depends_on("mpi", when="+mpi")
+    depends_on("caliper", when="+caliper")
+    depends_on("adiak", when="+caliper")
+    depends_on("hip", when="+rocm")
+    depends_on("cuda", when="+cuda")
+
```

Comment:
You will probably need this
```suggestion
    depends_on("c", type="build")
    depends_on("cxx", type="build")
```